### PR TITLE
Update extensions versions

### LINKED ISSUE #863: GA versions of worker extensions for latest trigger and binding extension versions
There is a batch of worker extensions that are still in preview and reference preview versions of the corresponding host extensions, but those host extensions are GA. These should be updated to GA versions, referencing the GA host extensions.

It may be appropriate to split these out into separate work items, but for brevity and getting it captured:

| Service | Worker Extension | Code link | Host Extension Specified | Latest Host Extension | Status | 
|-|-|-|-|-|-|
| Blobs | [5.0.0](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs/5.0.0) | [AssemblyInfo.cs](https://github.com/Azure/azure-functions-dotnet-worker/blob/aaf1cf7cd99fa075bc8700e988262f126eac61f1/extensions/Worker.Extensions.Storage.Blobs/src/Properties/AssemblyInfo.cs) | [5.0.0](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/5.0.0) | [latest](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs) - 5.0.0 at issue edit time| ✅ Aligned |
| Queues | [5.0.0](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.Storage.Queues/5.0.0) | [AssemblyInfo.cs](https://github.com/Azure/azure-functions-dotnet-worker/blob/aaf1cf7cd99fa075bc8700e988262f126eac61f1/extensions/Worker.Extensions.Storage.Queues/src/Properties/AssemblyInfo.cs) | [5.0.0](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/5.0.0) | [latest](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Storage.Queues) - 5.0.0 at issue edit time | ✅ Aligned |
| Event Hubs | [5.0.0-beta6](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.EventHubs/5.0.0-beta.6) | [AssemblyInfo.cs](https://github.com/Azure/azure-functions-dotnet-worker/blob/4b78a06fa34987769979bf4b2e335ce54482e455/extensions/Worker.Extensions.EventHubs/src/Properties/AssemblyInfo.cs) | [5.0.0-beta6](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.EventHubs/5.0.0-beta.6) | [latest](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.EventHubs) - 5.1.0 at issue edit time| Needs GA worker extension and specified host extension version bump |
| Service Bus | [5.0.0-beta6](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.ServiceBus/5.0.0-beta.6) | [AssemblyInfo.cs](https://github.com/Azure/azure-functions-dotnet-worker/blob/4b78a06fa34987769979bf4b2e335ce54482e455/extensions/Worker.Extensions.ServiceBus/src/Properties/AssemblyInfo.cs) | [5.0.0-beta6](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.ServiceBus/5.0.0-beta.6) | [latest](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.ServiceBus/) - 5.3.0 at issue edit time | Needs GA worker extension and specified host extension version bump |
| Event Grid | [3.0.0-beta3](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.EventGrid/3.0.0-beta.3) | [AssemblyInfo.cs](https://github.com/Azure/azure-functions-dotnet-worker/blob/24951c59af9182f58f96ad49356c4db69b3a44c3/extensions/Worker.Extensions.EventGrid/src/Properties/AssemblyInfo.cs) | [3.0.1](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.EventGrid/3.0.1) | [latest](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.EventGrid/) - 3.2.0 at issue edit time | Needs GA worker extension and specified host extension version bump |

The Storage meta-package is in fact GA: https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.Storage/ - this just references Blobs and Queues, though.

For Azure Tables, see https://github.com/Azure/azure-functions-dotnet-worker/issues/404

Cosmos DB also has a new major version of its host extension that remains in preview. It could probably stand an update as well to pull in the latest preview.

We should discuss options for bringing these into better alignment and preventing drift going forward. CC @JoshLove-msft

### PR DESCRIPTION
**Pull request by madelinegordon**
Updated extension versions of ServiceBus, EventGrid, and EventHubs. 

### Issue describing the changes in this PR

resolves #863

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
  * [x] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: extensions/Worker.Extensions.Storage.Tables/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - extensions/Worker.Extensions.Storage.Blobs/
+
+pr:
+  branches:
+    include:
+    - main
+    - release/*
+  paths:
+    include:
+    - extensions/Worker.Extensions.Storage.Blobs/
```
Comment (soninaren) [inline extensions/Worker.Extensions.Storage.Tables/ci.yml:17]: same as above

File: extensions/Worker.Extensions.Storage.Tables/src/Worker.Extensions.Storage.Tables.csproj [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs</RootNamespace>
+    <Description>Azure Blob Storage extensions for .NET isolated functions</Description>
```
Comment (soninaren) [inline extensions/Worker.Extensions.Storage.Tables/src/Worker.Extensions.Storage.Tables.csproj:6]: also here

File: extensions/Worker.Extensions.Storage.Tables/src/Worker.Extensions.Storage.Tables.csproj [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</RootNamespace>
+    <Description>Azure Table Storage extensions for .NET isolated functions</Description>
+    
+    <!--Version information-->
+    <VersionPrefix>1.0.0</VersionPrefix>
+
+    <!--Temporarily opting out of documentation. Pending documentation-->
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
```
Comment (fabiocav) [inline extensions/Worker.Extensions.Storage.Tables/src/Worker.Extensions.Storage.Tables.csproj:12]: It looks like the attributes have been documented. Should we still be opting out here? What else are we missing?

**Review by fabiocav — COMMENTED** [review]
Minor comments. If we continue to iterate on the tables extension, it might be helpful to separate this into its own PR so the other items are not blocked.

File: extensions/Worker.Extensions.ServiceBus/src/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -3,4 +3,4 @@
 
 ﻿using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
-[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.ServiceBus", "5.0.0-beta.6")]
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.ServiceBus", "5.3.0")]
```
Comment (kshyju) [inline extensions/Worker.Extensions.ServiceBus/src/Properties/AssemblyInfo.cs:6]: Looks like 5.4.o is the latest version of this package. https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.ServiceBus/

Comment (kshyju) [reply extensions/Worker.Extensions.ServiceBus/src/Properties/AssemblyInfo.cs:6]: It was released yesterday 😆

Comment (madelinegordon) [reply extensions/Worker.Extensions.ServiceBus/src/Properties/AssemblyInfo.cs:6]: Good catch, I'll update this.

**Review by fabiocav — APPROVED** [review]

**Review by soninaren — APPROVED** [review]

**Comment by kshyju** [comment]
[Microsoft.Azure.Functions.Worker.Extensions.ServiceBus version 5.4.0](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.ServiceBus/5.4.0) is now available in nuget gallery.

**Comment by kshyju** [comment]
[Microsoft.Azure.Functions.Worker.Extensions.EventHubs 5.1.0](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.EventHubs/5.1.0) is now available in nuget gallery.

**Pull request merged**